orangecontrib/text/preprocess.py

```diff
@@ -57,8 +57,11 @@ def __init__(self, lowercase=True, stop_words=None,
                 of expected values/types.
         """
         # Tokenizer.
-        self.tokenizer = word_tokenize
-        if use_twitter_tokenizer:
+        if tokenizer == 'default':
+            self.tokenizer = word_tokenize
+        elif tokenizer == 'no_punct':
+            self.tokenizer = RegexpTokenizer(r'\w+').tokenize
+        elif tokenizer == 'twitter':
             self.tokenizer = None   # TODO: Change when twitter is available.
```

Comment:
Will do. The only thing is that now the tokenizers are referenced as strings when building a preprocessor. But I guess that is fine until we agree on a better sollution.